packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -598,6 +600,29 @@ export default class Meeting extends StatelessWebexPlugin {
      * @memberof Meeting
      */
     this.shareStatus = SHARE_STATUS.NO_SHARE;
+    /**
+     * @instance
+     * @type {Boolean}
+     * @readonly
+     * @private
+     * @memberof Meeting
+     */
+    Object.defineProperty(this, 'isLocalShareLive', {
+      get: () => {
+        const {shareTransceiver} = this.mediaProperties.peerConnection;
+        const shareDirection = shareTransceiver?.direction;
+        const trackReadyState = shareTransceiver?.sender?.track?.readyState;
+        const activeShare = trackReadyState === LIVE;
+        const offersToSendData = shareDirection === SENDRECV;
```

Comment:
shouldn't offersToSendData be also set to true if shareDirection is "sendonly"? it's a minor thing, because in reality most apps would always want to receive shares too, but in theory we could have an app using SDK that doesn't care about remote presentations and use "sendonly"